engine/src/mcstring.cpp

```diff
@@ -1077,6 +1079,8 @@ const struct { const char *cstring; MCNameRef *name_var; } kInitialNames[] =
     { "(Message)", &MCN_font_message },
     { "(Tooltip)", &MCN_font_tooltip },
     { "(System)", &MCN_font_system },
+	
+	{ "systemAppearanceChanged", &MCM_system_appearance_changed },
```

Comment:
I suggest putting this with the other all-platform engine message names - after updateVar?